shared/team-building/index.tsx

```diff
@@ -86,10 +86,11 @@ const ContactsBanner = ({onRedoSearch}: {onRedoSearch: () => void}) => {
   const onLater = React.useCallback(() => dispatch(SettingsGen.createImportContactsLater()), [dispatch])
   // Redo search if # of imported contacts changes
   React.useEffect(() => {
-    if (prevNumContactsImported !== undefined && prevNumContactsImported !== numContactsImported) {
+    if (prevNumContactsImported !== numContactsImported) {
```

Comment:
We should keep `prevNumContactsImported !== undefined &&`, it avoids a false positive on mount.

Reply:
Doesn't it always start as undefined until we import the contacts? I'll put it back in and test, I guess.

Reply:
On mount `numContactsImported === null` and `prevNumContactsImported === undefined`
https://github.com/keybase/client/blob/1227eeb259d67a5bd35ceef507be69370ca60d6b/shared/constants/settings.tsx#L90-L95

Reply:
🤯

Reply:
@buoyad got it, thanks.